plugins/webp-uploads/picture-element.php

```diff
@@ -168,3 +168,20 @@ function webp_uploads_wrap_image_in_picture( string $image, string $context, int
 		$image
 	);
 }
+
+/**
+ * Adds CSS to make Lightbox trigger BUTTON visible on an Image lightbox when Picture Element is enabled.
+ *
+ * @since 2.4.0
+ * 
```

Comment:
The version we'll supply when creating the release:

```suggestion
 * @since n.e.x.t
```